ganga/GangaCore/Lib/Condor/Condor.py

```diff
@@ -797,4 +797,37 @@ def timedetails(self):
     updateMonitoringInformation = \
         staticmethod(updateMonitoringInformation)
 
+    def peek(self, filename=None, command=None):
```

Comment:
So apparently `condor_tail` allows you to peak any filename in the `transfer_output_files` list. I wonder if it is worth giving the option to users to specify a filename here.

Also I would fill in the command to include some options so maybe a user can set the buffer length or peak the stderr